jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -1735,22 +1735,64 @@ static MethodHandle filterArgument(MethodHandle target, int pos, MethodHandle fi
 		return Lookup.PUBLIC_LOOKUP;
 	}
 	
-	/*[IF Sidecar19-SE]*/
+	/*[IF Sidecar19-SE-B175]*/
 	/**
-	 * Return a MethodHandles.Lookup object that is only able to access <code>private</code> members.
+	 * Return a MethodHandles.Lookup object with full capabilities including the access 
+	 * to the <code>private</code> members in the requested class
 	 * 
 	 * @param targetClass - the requested class containing private members
 	 * @param callerLookup - a Lookup object specific to the caller
 	 * @return a MethodHandles.Lookup object with private access to the requested class
+	 * @throws NullPointerException - if any of the arguments is null
```

Comment:
Pls be specific about the argument, `targetClass` or `callerLookup`.

Reply:
Updated.